util-logging/src/main/scala/com/twitter/logging/FileHandler.scala

```diff
@@ -166,8 +166,10 @@ class FileHandler(
   }
 
   private def openStream(): OutputStream = {
-    val dir = new File(filename).getParentFile
+    val file = new File(filename)
+    val dir = file.getParentFile
     if ((dir ne null) && !dir.exists) dir.mkdirs
+    bytesWrittenToFile = if (file.exists()) file.length() else 0
```

Comment:
The bytesWrittenToFile is set by openStream() which is called inside a synchronized block under openLog(). That preserves it's synchronized lock.

The purpose of removing it from openLog and adding it to openStream is to make sure that as and when the log file opened up or created, the file size is set at that moment.